packages/twenty-server/src/utils/generate-secret.ts

```diff
@@ -0,0 +1,14 @@
+import { createHash } from 'crypto';
+
+if (!process.env.APP_SECRET) {
```

Comment:
We should access env var through the dedicate service not directly through process.env ; you should probably do this in AuthService

Reply:
Got it. I'll see that.